File: POO_Semana1/Pytho_Semana1/Ejercicio04/AnualSalary.py
class Employ:
    Name = ""
    Salary = 0


    def CalculateAnualSalary(self):
        return self.Salary * 12

employ1 = Employ()
employ2 = Employ()
employ1.Name = "Jeremy"

employ1.Salary = 7800

total1 = employ1.CalculateAnualSalary()

print("Empleado: ",employ1.Name)
print("El salario anual es: ", total1)

employ2.Name = "Eduard"

employ2.Salary = 8500

total2 = employ2.CalculateAnualSalary()

print("Empleado: ",employ2.Name)
print("El salario anual es: ", total2)

File: POO_Semana1/Pytho_Semana1/Ejercicio04/AnualSalary.java
class Employ{
    String Name = "";
    int Salary = 0;

    int CalculateAnualSalary(){
        return Salary * 12;
    }
}


public class AnualSalary{
    public static void main(String[] args){
        Employ employ1 = new Employ();
        Employ employ2 = new Employ();

        employ1.Name = "Jeremy";
        employ1.Salary = 8600;
        int total1 = employ1.CalculateAnualSalary();

        employ2.Name = "Eduard";
        employ2.Salary = 8700;
        int total2 = employ2.CalculateAnualSalary();
        System.out.println("---"+employ1.Name+"---");

        System.out.println("El Salario anual es: $" + total1);
        System.out.println("---" + employ2.Name + "---");

        System.out.println("El Salario anual es: $" + total2);



    }
}